src/main/java/frc/robot/Robot.java

```diff
@@ -31,9 +62,56 @@ public class Robot extends TimedRobot {
   public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
-    m_robotContainer = new RobotContainer();
+    configureButtonBindings();
     
+    // starts the auto selector
+    autoChooser.setDefaultOption("Auto Balance Mobile", new AutonomousBalanceMobile(m_driveSubsystem, m_intakeSubsystem, m_bucketSubsystem, gyro));
+    autoChooser.addOption("Auto Grab", new AutonomousGrab(m_driveSubsystem, m_intakeSubsystem, m_bucketSubsystem));
+    autoChooser.addOption("Auto No Mobile", new AutonomousBalanceNoMobile(m_driveSubsystem, m_intakeSubsystem, m_bucketSubsystem, gyro));
+    autoChooser.addOption("doNothing", new InstantCommand());
+    autoChooser.addOption("Dump Do Nothing", new AutonomousDumpDoNothing(m_driveSubsystem, m_intakeSubsystem, m_bucketSubsystem));
+  
+    SmartDashboard.putData("autos: ", autoChooser);
+
+
+
+
+    //start cameraServer
+    CameraServer.startAutomaticCapture();
+    CameraServer.startAutomaticCapture();
```

Comment:
do we need 2 lines that start the camera's capture?

Reply:
idk but i will test it on fri if i have time